app/javascript/packs/homePageFeed.jsx

```diff
@@ -150,7 +150,7 @@ export const renderFeed = async (timeFrame, afterRender) => {
     }
 
     return (
-      <div>
+      <Fragment>
```

Comment:
Switched to a Fragment here to eliminate the filler div and make querying the children articles more straightforward